### PR TITLE
Reservation test

### PR DESCRIPTION
**Pull request by themonster2015**
This PR adds the following changes to `dev` 👇 :

- Create tests for reservation model

### CODE REVIEW

File: spec/models/reservation_spec.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+  subject do
+    Reservation.create(user_id: @user.id, destination_id: @destination.id, startingDay: '01/01/2015',
+                       endingDay: '01/01/2015')
+  end
+
+  it 'create a valid destination object' do
+    expect(subject).to be_valid
+  end
+
```
Comment (AleksandarIvezic) [inline spec/models/reservation_spec.rb]: You don't need to make this mocking variables when you using shoulda gem. So you can remove this piece of code. It's unnecessary

**Review by AleksandarIvezic — CHANGES_REQUESTED** [review]
Hi @themonster2015,
 Great job so far. 
However, there are still some things to be improved.
Please check the comments below.

Happy Codding!!! Cheers!!! 🥂

File: spec/models/reservation_spec.rb (hunk 1 above)
Comment (themonster2015) [reply spec/models/reservation_spec.rb]: thanks @ShinobiWarior for your suggestion. I agree and have fixed it.

**Comment by danmainah** [comment]
Good work @themonster2015  
All test are running and you have implemented changes requested 

Your project is Approved ✔✔✔
🙌🙌🙌
Cheers and Happy Coding

**Pull request merged**